frontend/src/components/teacher/student-management/classroom-summary/AddOrEditClassroomModal.tsx

```diff
@@ -153,7 +207,7 @@ const AddClassroomModal = ({
         <ModalContent borderRadius="12px" maxW="80vw" p={2}>
           <ModalHeader>
             <Text color="grey.400" textStyle="subtitle1">
-              Add Classroom
+              {classroomId ? "Edit Classroom Information" : "Add Classroom"}
```

Comment:
nit: (to be consistent, even if it differs from design)
```suggestion
              {classroomId ? "Edit Classroom" : "Add Classroom"}
```

Reply:
Good point, I'll update this in both places it occurs